.github/workflows/helm_chart_release.yaml

```diff
@@ -8,8 +8,13 @@ on:
     paths:
       - "charts/**"
 
+permissions:
+  contents: read
```

Comment:
Hm what's the effect of this if the only job bumps its permissions to `write`? Just covering our bases when/if another `job` is added?

Reply:
Yes, the top-level permissions tag will only provide minimum access which can be overridden by job-level permissions tag. It is placed as a security measure.
Adding reference for this: [Assigning permissions to jobs](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs)